docs/intro/getting-started-with-mkdocs.rst

```diff
@@ -0,0 +1,66 @@
+Getting Started with MkDocs
+===========================
+
+MkDocs is a documentation generator that focuses on speed and simplicity.
+It has many great features including:
+
+* Preview your documentation as you write it
+* Easy customization with themes and extensions
+* Writing documentation with Markdown
+
+.. note::
+
+    MkDocs is a great choice for building technical documentation.
+    However, Read the Docs also supports :doc:`Sphinx </intro/getting-started-with-sphinx>`,
+    another tool for writing and building documentation.
+
+
+Quick start
+-----------
+
+Assuming you have Python already, `install MkDocs`_:
+
+.. sourcecode:: bash
+
+    $ pip install mkdocs
```

Comment:
I'm not sure how this is rendered, but maybe dropping the `$` makes easier to copy/paste

Reply:
It doesn't really render any different from the [existing getting started guide](https://docs.readthedocs.io/en/latest/getting_started.html#in-restructuredtext). The only difference is that any shell builtins (eg. `echo`, `cd`, etc.) are syntax highlighted. It also is more explicit.

There's a few places in our docs that use the pattern of `$ shellcommand` so I'm hesitant to change it without changing it everywhere.

Reply:
I generally dislike the `$` for the reason mentioned (hard to copy/paste), but don't feel strongly.